control/plugin/execution.go

```diff
@@ -131,6 +132,25 @@ func (e *ExecutablePlugin) Run(timeout time.Duration) (Response, error) {
 				}).Debug(stdOutScanner.Text())
 			}
 		}
+
+		if err := stdOutScanner.Err(); err != nil {
+			if err == bufio.ErrTooLong {
+				reader := bufio.NewReader(e.stdout)
+				log, _, _ := reader.ReadLine()
```

Comment:
I don't see any special reasons to use `Readline()` instead of `ReadString('\n')`. I changed `Readline()`to `ReadString('\n')` to avoid parsing to string.